examples/prometheus-operator/prometheus.yaml

```diff
@@ -0,0 +1,78 @@
+# more details can be found
+# https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.Prometheus
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: jobset-metrics
+  namespace: jobset-system
+spec:
+  serviceAccountName: prometheus-jobset
```

Comment:
Does this service account need to be created before we apply the Prometheus CR? I see you create it lower in this file - have you tested this manually?

Reply:
I split into rbac.yaml

Reply:
Why not just put the service account and RBACs above the Prometheus CR in the same file?

Reply:
done